packages/origin-backend/src/pods/organization/organization.service.ts

```diff
@@ -169,6 +169,7 @@ export class OrganizationService {
         }
 
         await this.userService.removeOrganization(memberId);
+        await this.userService.changeRole(memberId, Role.OrganizationAdmin);
```

Comment:
Not sure if this is a proper fix. Users should have no role set if they are not a part of the organization.

Maybe adding a new role `Role.None` would be the way to go, and then adjust the backend endpoints if needed.

Reply:
This default as OrganizationAdmin is used during user registration as well, it's not the best/cleanest but something we already been relying on.

Reply:
I agree. This "fix" was done according to the existing roles.